frontends/bsd/src/screens/admin_actions_screen.tsx

```diff
@@ -145,8 +151,22 @@ export function AdminActionsScreen({
               <p>
                 <SetClockButton>Update Date and Time…</SetClockButton>
               </p>
+              {!hasExportedBackupForAllBatches && (
+                <Text italic muted>
+                  Export a backup in order to delete election data.
+                </Text>
+              )}
+              {hasExportedBackupForAllBatches && exportedAt && (
+                <Text italic muted>
+                  Election data backed up at {shortDateTime(exportedAt)}
+                </Text>
+              )}
               <p>
-                <Button danger onPress={toggleIsConfirmingUnconfigure}>
+                <Button
+                  danger
+                  disabled={!hasExportedBackupForAllBatches}
+                  onPress={toggleIsConfirmingUnconfigure}
+                >
                   Delete Election Data from VxCentralScan…
                 </Button>
```

Comment:
The text above the button is equally close to the button above it too. We should more closely associate the text to the button it's related to.

I haven't run the code below to ensure it works, but it should:
- Place the text to the right of the button.
- Update copy to be more specific. (Though now realizing that "Export Backup" should probably be changed to something more descriptive which doesn't contain two closely related verbs/tems. 🙃 )
- Use warning style text.
- Use a warning icon.

I've removed the italic style as I'd like to save that for adding emphasis to content next to non-italicized content.

```suggestion
                <Button
                  danger
                  disabled={!hasExportedBackupForAllBatches}
                  onPress={toggleIsConfirmingUnconfigure}
                >
                  Delete Election Data from VxCentralScan…
                </Button>{' '}
                {!hasExportedBackupForAllBatches && (
                  <Text as="span" warning warningIcon>
                    You must "Export Backup" before you may delete election data.
                  </Text>
                )}
                {hasExportedBackupForAllBatches && exportedAt && (
                  <Text as="span" muted>
                    Election data was last backed up at {shortDateTime(exportedAt)}
                  </Text>
                )}
```

Reply:
Oof. Perhaps place a `<br />` before the `<span>`? so that  the text is under the button and more tightly associated with the button? (sorry for the delay in response)